scripts/tables.R

```diff
@@ -4,8 +4,8 @@
 
 
 # add project specific variables ------------------------------------------
-filename_html <- 'Skeena2022'
-repo_name <- 'fish_passage_skeena_2022_reporting'
```

Comment:
I successfully tested a better way to do this which is to add params.  Template has them added now here https://github.com/NewGraphEnvironment/fish_passage_template_reporting/blob/a86baeefd4ffa3632d7b6652454450746f1cbc59/index.Rmd#L23 with an example of how they are called here https://github.com/NewGraphEnvironment/fish_passage_template_reporting/blob/a86baeefd4ffa3632d7b6652454450746f1cbc59/2100-session-info.Rmd#L5

tables.R has a couple spots that object was called so we will need to use the `param` version instead. Its great that now we can use the `params` outside of `tables.R` . You will start to see `param${this_or_that}` in quite a few places and it should help reduce hand bombing and subsequent missed spots where the hand bombing was previously needed.

Reply:
nice I like how thats done!